internal/server/kong/ws/server.go

```diff
@@ -119,6 +122,13 @@ func (h websocketHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		loggerWithNode.Error("failed to upgrade websocket connection", zap.Error(err))
 		return
 	}
+	dpConnectionDuration := time.Since(dpConnectStartTime).Milliseconds()
+	metrics.Histogram("data_plane_time_to_connect",
```

Comment:
Related to the other comments, can we use seconds here & rename to `data_plane_connect_duration_seconds`.

Also, should we rename this to `data_plane_websocket_connect_duration_seconds` to be a bit more specific?

Reply:
I've seen us use a "protocol": "ws" metric tag in other places. I don't have an opinion either way.

Reply:
> I've seen us use a "protocol": "ws" metric tag in other places. I don't have an opinion either way.
At least right now, there's only one protocol here, as it's all websockets anyway (for both wRPC & the older WS protocol). We don't need to rename it to be a bit more specific, was mostly just a question whether we should.